src/test/java/se/sundsvall/parkingpermit/businesslogic/worker/execution/CreateAssetTaskWorkerTest.java

```diff
@@ -65,6 +82,5 @@ void executeThrowsException() {
 		verify(failureHandlerMock).handleException(externalTaskServiceMock, externalTaskMock, thrownException.getMessage());
 		verify(externalTaskMock).getId();
 		verify(externalTaskMock).getBusinessKey();
-		verifyNoInteractions(camundaClientMock);
```

Comment:
Change test to throw exception during call to `getErrand(externalTask)` and `verify(externalTaskServiceMock, never()).complete(any())` and 
`verifyNoInteractions(citizenAssetsServiceMock)`?